pkg/cri/cri.go

```diff
@@ -166,3 +174,28 @@ func getNRIPlugin(ic *plugin.InitContext) (nri.API, error) {
 
 	return api, nil
 }
+
+// Get the Network plugin and verify its type.
+func getNetworkPlugin(ic *plugin.InitContext) (compat.API, error) {
+	const (
+		pluginType = plugin.NetworkPlugin
+		pluginName = "cni"
+	)
+
+	if os.Getenv("ENABLE_NETWORK_SRV") == "" {
```

Comment:
I believe that this is inspired by ENABLE_CRI_SANDBOXES. But do you think we cannot make this stable enough before releasing 1.7.0? My worry is that nobody use this if that is opt-in.